### PR TITLE
first design with filtrify and asset layout

### PR DESCRIPTION
**Pull request by ssontag55**
Don't merge    :)  just FYI

### CODE REVIEW

**Comment by birdage** [comment]
@ssontag55 any reason for the filtrify source in the code? that not a bower install?

**Comment by ssontag55** [comment]
\assets\list
\events\list

Click on an event for editing/creating and go to
\events\list\<id>

adding new now

**Comment by ssontag55** [comment]
@DanielJMaher actually, now its ready...

File: ooiui/config/config.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ COMMON: &common
   SERVICES_URL: http://localhost:4000
   DEBUG: False
   CACHE_TYPE: simple
-  UI_API_KEY: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-  SECRET_KEY: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+  UI_API_KEY: 'ccdf5de08ac74855bda3e7e309d871e5'
```
Comment (oceanzus) [inline ooiui/config/config.yml]: Not cool!

File: ooiui/static/js/views/asset_management/AssetView.js [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+                    });
+                    
+                    //reset
+                    selectedInstrument = [];
+                    self.clearform();
+                }
+            }
+        });
+  
+        //add events for DMS changes
```
Comment (oceanzus) [inline ooiui/static/js/views/asset_management/AssetView.js]: Sexy!

**Comment by oceanzus** [comment]
Fix the config.yml and it's good to go.

File: ooiui/config/config.yml (hunk 1 above)
Comment (ssontag55) [inline ooiui/config/config.yml]: Son of a

Pleading the 5th

Stephen

**Pull request merged**